Makefile

```diff
@@ -34,6 +38,18 @@ test-install-onnx:
 		--backend onnx                      \
 		--junitxml ${JUNITXML_PATH}
 
+ONNX_E2E_PTQ_SIZE ?= 300
+ONNX_E2E_PTQ_EVAL_SIZE ?=
+ONNX_E2E_OPTIONS=--model-dir ${MODELS_DIR} --data-dir ${DATA_DIR}  \
+				 --output-dir ${OUTPUT_DIR} --anno-dir ${ANNO_DIR} \
+                 --junitxml ${JUNITXML_PATH} --ptq-size ${ONNX_E2E_PTQ_SIZE} \
+                 --eval-size ${ONNX_E2E_PTQ_EVAL_SIZE}
+
+test-e2e-ptq:
```

Comment:
My general comment that I would like to avoid "super" makefile which includes all possible targets with many input arguments. Could you propose the solution of this problem?

Reply:
I think running e2e does not belong as a makefile target at all, it has too many parameters for that which must be forwarded via clunky make CLI. I think that if we want to decouple CI-defined, stable parameter set such as dataset folders from the set of parameters that may be changed frequently by developers, so that devs can make faster changes without involving a change in CI repo, we could make a wrapper script such as run_e2e_onnx.py, which will accept only the CI-dependent args (i.e. dataset folder) and then forward them to a pytest call via subprocess.